sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/DescribeTableSuite.scala

```diff
@@ -176,7 +176,7 @@ class DescribeTableSuite extends command.DescribeTableSuiteBase
     }
   }
 
-  test("describe extended (formatted) a column without col stats") {
+  test("SPARK-39859: describe extended (formatted) a column without col stats") {
```

Comment:
@yaooqinn I'm not using the new JIRA id, is it appropriate to be added here too?

Reply:
Please create a new JIRA ticket to track, SPARK-39859 is not suitable here or in the title.